include/deal.II/lac/vector.templates.h

```diff
@@ -48,6 +48,23 @@
 
 DEAL_II_NAMESPACE_OPEN
 
+/**
+ * Helper struct to tell us if we can use SIMD instructions for the given @p
+ * Number type.
+ */
+template <typename Number>
+struct UseSimdFor
+{
+  static const bool value = true;
+};
+
+// disable long double SIMD
+template <>
+struct UseSimdFor<long double>
+{
+  static const bool value = false;
+};
```

Comment:
We can. I think the conclusion was that there are no ```long double``` SIMD instructions anyways.